src/Components/Profiles/Modal/modal.module.css

```diff
@@ -0,0 +1,36 @@
+.container{
+  position:absolute;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 120px;
+  width: 100%;
+  background-color: rgba(247, 233, 233, 0.5);
```

Comment:
please, just use the styles I am using here https://github.com/BaSD-2021/Esteban-MindSet-app/pull/3/files#diff-ed523f7f05eb0244fb2951825b943d3c5a120bad0e90f877acee0683ff0ca377R1-R9

Reply:
Styles changed